### DISCUSSION TITLE
The Vercel build fails when run from a commit but not when run using the CLI locally

### DISCUSSION THREAD

**Discussion by vemec** [vercel/community#2623] [Help]
### Summary

Hello how are you? I hope everyone is well

This is the error that appears in the build generated by a git commit
```shell
- info Loaded env from /vercel/path0/.env
--
11:39:08.069 | - info Creating an optimized production build...
11:39:26.149 | Failed to compile.
11:39:26.149 |  
11:39:26.149 | ./src/app/(authenticated)/_components/Layout/Header/index.js
11:39:26.150 | Module not found: Can't resolve '@/uiComponents/Button'
```

But when you locally run the `vercel` CLI command the build and deploy are generated without any problems.
```shell
vercel

Vercel CLI 30.0.0
🔍  Inspect: https://vercel.com/vemec/xxxxxx/DKd2Arg4TFY4siRESDTCkDi7gtd6 [4s]
✅  Preview: https://xxxxxx.vercel.app [53s]
📝  To deploy to production (xxxxxx.vercel.app), run `vercel --prod`
```

![Screenshot 2023-05-26 at 12 04 55](https://github.com/vercel/community/assets/839090/ad83a3cc-36dc-47af-895f-af0f0d8df3b6)

Any idea what might be going on?

Thank you very much in advance

### Steps to Reproduce

commit and fail, run vercel locally works

**Comment by amyegan** [comment]
Hi @vemec. This problem isn't happening in my projects. Please share a minimal reproduction, or your repo if it is public, so I can get a better look at what is causing the problem.

**Comment by z4nr34l** [comment]
Could you bring us more details?
I consider as helpful those informations:
1. Repository Access;
2. Vercel build logs (from deployment with this issue);
3. Runtime logs from problematic request;

The more - the better for receiving quick help.

You can also try to reproduce this problem on free [CodeSandbox](https://codesandbox.io/) and post URL here when it's done.

**Comment by YKalashnikov** [comment]
Hello @vemec are you still facing that issue?  Seems like im on the same page
